lib/google_tasks/service.rb

```diff
@@ -133,7 +133,7 @@ def tasklist
 
     # In case a reclaim title is present, match the title
     def friendly_titles_match?(google_task, task)
-      matcher = /\A(?<title>.+)\s*(?<addon>.*)\Z/i
+      matcher = /\A(?<title>.+)\s(?<time clause>\(for\s.*[minutes|hours]\))(?<addon>.*)\Z/i
```

Comment:
I think the issue here is that the `<title>` capture is too greedy - a better fix is probably to just add the title to the matcher itself - the current `<addon>` matcher _should_ catch the "for x minutes" string. 

Also, this adds an extra space to the matcher, so this will probably cause false negatives for tasks that don't have an estimated time.  
```suggestion
      matcher = /\A(?<title>#{task.title.strip})\s*(?<addon>.*)\Z/i
```
I think this -^ will prevent the greedy capture.

Reply:
You may be right. FWIW, I think pushing that Reclaim estimated time into Google Tasks is a great feature. 

Although, it will be moot if Reclaim becomes a first-class citizen.... At which point, it may make sense to push meta-data like estimated time into the details or notes, for visibility in the Google Tasks UI.